base_img/hooks/ssh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+run_hook() {
+    mkdir /dev/pts
+    mount -t devpts /dev/pts /dev/pts
+    dropbear -E
+    sshd_running=1
```

Comment:
what does that line do?

Reply:
In general, the hook functions define this variable so that other init-related functions can readily check if the service has been launched without having to resort to something like pgrep. We don't specifically use this variable, currently, but it is defined for consistency and for potential future use.